src/ansible_navigator/configuration_subsystem/parser.py

```diff
@@ -1,20 +1,18 @@
 """Build the argument parser
 """
 
+from argparse import SUPPRESS
```

Comment:
This is weird... Is it sorted like this because it's all-uppercase?

Reply:
yeah, I assumed so when I saw it